docs/logging.rst

```diff
@@ -109,7 +102,7 @@ thing as the dictionary based configuratio above:
     filters = request_id
 
     [formatter_json]
-    class = dockerflow.logging.JsonLogFormatter
+    class = dockerflow.logging.MozlogFormatter
```

Comment:
Is this necessary if we don't leverage it in the handlers section?

Reply:
For file config, the file *must* have a `loggers`, `handlers`, and `formatters` section. We could do something like we do in the tests now:

```
    [formatters]
    keys =
```

But it feels okay to have one place in the docs that shows you can use the handler and still specify a custom formatter, if you wanted. 

Also maybe we should just deprecate this `fileConfig` documentation, since it's [more or less deprecated](https://docs.python.org/3/library/logging.config.html#configuration-file-format) in Python:

> The [fileConfig()](https://docs.python.org/3/library/logging.config.html#logging.config.fileConfig) API is older than the [dictConfig()](https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig) API and does not provide functionality to cover certain aspects of logging. For example, you cannot configure [Filter](https://docs.python.org/3/library/logging.html#logging.Filter) objects, which provide for filtering of messages beyond simple integer levels, using [fileConfig()](https://docs.python.org/3/library/logging.config.html#logging.config.fileConfig). If you need to have instances of [Filter](https://docs.python.org/3/library/logging.html#logging.Filter) in your logging configuration, you will need to use [dictConfig()](https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig). Note that future enhancements to configuration functionality will be added to [dictConfig()](https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig), so it’s worth considering transitioning to this newer API when it’s convenient to do so.

Reply:
> Also maybe we should just deprecate this fileConfig documentation, since it's [more or less deprecated](https://docs.python.org/3/library/logging.config.html#configuration-file-format) in Python:

OH TIL !